Source/OxyPlot.ImageSharp/OxyPlot.ImageSharp.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
Should we add `<LangVersion>8</LangVersion>`? Then we could use those tasty switch expressions ;)

Reply:
Personally I deplore switch expressions, but scoped using is nice yo have ;)

Reply:
I caved: langversion 8 it shall be ;)